src/main/java/org/dominokit/domino/ui/forms/ValueBox.java

```diff
@@ -255,13 +255,15 @@ public T setRightAddon(Element rightAddon) {
     public T removeRightAddon() {
         if (nonNull(rightAddon)) {
             rightAddonContainer.removeChild(rightAddon);
```

Comment:
I think the solution the issue should be by replacing `rightAddonContainer.removeChild(rightAddon);` with `rightAddon.remove()`